build-system/tasks/e2e/describes-e2e.js

```diff
@@ -579,6 +611,35 @@ class EndToEndFixture {
       await controller.dispose();
     }
   }
+
+  /**
+   * Translate relative fixture specs into localhost test URL.
+   * @param {!TestSpec} spec
+   * @return {!TestSpec}
+   */
+  setTestUrl(spec) {
+    // TODO(rcebulko): See if it's possible to condense/reorg some test fixtures
+    // and have all e2e fixtures in one folder. Allowing this split for now so
+    // that, if fixtures under `manual` are moved, tests using normal e2e fixture
+    // directory don't need to be updated. Support for `testUrl` should
+    // eventually be removed entirely.
+    let {testUrl} = spec;
+    const {fixture, manualFixture} = spec;
+    if (!!testUrl + !!fixture + !!manualFixture > 1) {
+      console.log(spec);
+      throw new Error(
+        'Only one of [testUrl, fixture, manualFixture] may be specified'
+      );
+    }
+
+    if (fixture) {
+      testUrl = `http://localhost:8000/test/fixtures/e2e/${fixture}`;
+    } else if (manualFixture) {
+      testUrl = `http://localhost:8000/test/manual/${manualFixture}`;
```

Comment:
There is no usage where `testUrl` in an e2e test is not one of these two

Reply:
Update: see https://github.com/ampproject/amphtml/issues/32177 for eventual next step